config/crd/bases/compliance.openshift.io_compliancescans.yaml

```diff
@@ -190,6 +190,17 @@ spec:
                   for a specific rule. Note that when leaving this empty, the scan
                   will check for all the rules for a specific profile.
                 type: string
+              scanLimits:
+                additionalProperties:
+                  anyOf:
+                  - type: integer
+                  - type: string
+                  pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
+                  x-kubernetes-int-or-string: true
+                description: ScanLimits allows to set the resource limits that the
+                  scan pods are allowed to use. By default, compliance operator will
+                  use sensible defaults
```

Comment:
Do we, or should we, document defaults here?

Reply:
I added a comment to the type itself, but for some reason it doesn't seem to propagate to the CRDs..ugh..

Reply:
becuse it's `make manifests` /facepalm